crates/services/importer/src/importer.rs

```diff
@@ -223,12 +232,55 @@ where
             .seal_block(&block_id, &result.sealed_block.consensus)?
             .should_be_unique(&expected_next_height)?;
 
+        // Update the total tx count in chain metadata
+        let total_txs = db_after_execution
+            // Safety: casting len to u64 since it's impossible to execute a block with more than 2^64 txs
+            .update_tx_count(result.sealed_block.entity.transactions().len() as u64)?;
+
         db_tx.commit()?;
 
-        tracing::info!("Committed block");
+        // update the importer metrics after the block is successfully committed
+        IMPORTER_METRICS.total_txs_count.set(total_txs as i64);
```

Comment:
Is a signed type necessary here? Looking at the [documentation](https://docs.rs/prometheus-client/latest/prometheus_client/metrics/gauge/struct.Gauge.html#using-atomicu64-as-storage-and-f64-on-the-interface), `i64` is the default, but it appears that this can be configured to a signed type if we so choose.

Reply:
It only uses the AtomicU64 if the gauge is an f64